src/plotting/plot.component.tsx

```diff
@@ -96,43 +119,100 @@ export const Plot = (props: FullPlotProps) => {
             data: { fill: '#e31a1c' },
           }}
           data={data}
-          x="timestamp"
-          y="shotnum"
+          x={XAxis}
+          y={YAxis}
           size={type === 'line' ? 2 : 3}
-          labels={({ datum }) => `(${datum._x}, ${datum._y})`}
+          labels={({ datum }) => {
+            const formattedXLabel = formatTooltipLabel(
+              datum._x,
+              XAxisSettings.scale
+            );
+            const formattedYLabel = formatTooltipLabel(
+              datum._y,
+              YAxesSettings.scale
+            );
+            return `(${formattedXLabel}, ${formattedYLabel})`;
+          }}
           labelComponent={<VictoryTooltip />}
         />
       </VictoryChart>
     </div>
   );
 };
 
-interface PlotProps {
-  title: string;
-  type: PlotType;
-  XAxisSettings: AxisSettings;
-  YAxesSettings: AxisSettings;
-}
+export const getFormattedAxisData = (
+  record: Record,
+  scalarChannels: FullScalarChannelMetadata[],
+  axisName: string
+): number => {
+  let formattedData = NaN;
+
+  switch (axisName) {
+    case 'timestamp':
+      formattedData = new Date(record.metadata.timestamp).getTime();
```

Comment:
Doesn't this convert a `timestamp` - which is already a number representing seconds from epoch, into a JS Date which is then converted into an int representing seconds from epoch (via `getTime`)? Why do this at all?

Reply:
Timestamp is currently in this format:
![image](https://user-images.githubusercontent.com/71134574/188835640-980b791f-fd9c-4b53-af30-84f3ae84858c.png)

It's capable of handling milliseconds too but at the moment this is the format it's in. I would have thought Unix timestamps would be a better measurement too

Reply:
Oh, I didn't realise that in the metadata they're actual time strings. I guess I was used to our mock data where timestamp was a unix timestamp. No worries then!